src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -1,13 +1,12 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
WinRM based remoting session abrupt disconnect.Error occurred in Describe block
```powershell
Safe handle has been closed.
Object name: 'SafeHandle'.
At D:\a\1\s\test\powershell\engine\Remoting\RemoteSession.Disconnect.Tests.ps1:60 char:35
```